features/cellular/TESTS/socket/udp/main.cpp

```diff
@@ -31,7 +31,7 @@
 #endif
 
 #if defined(TARGET_ADV_WISE_1570) || defined(TARGET_MTB_ADV_WISE_1570)
-#error [NOT_SUPPORTED] target MTB_ADV_WISE_1570 is too unstable for network tests, IoT network is unstable
+//#error [NOT_SUPPORTED] target MTB_ADV_WISE_1570 is too unstable for network tests, IoT network is unstable
```

Comment:
is this intentionallly, is yes then remove also #ifdef's

Reply:
that was unintentional, fixing